lib/assets/javascripts/attachinary.js.coffee

```diff
@@ -12,8 +12,8 @@
             <li>
               <% if(files[i].resource_type == "raw") { %>
                 <div class="raw-file"></div>
-              <% } else if (files[i].format == "mp3") { %>
-                <audio src="<%= $.cloudinary.url(files[i].public_id, { "version": files[i].version, "resource_type": 'video', "format": 'mp3'}) %>" controls />
+              <% } else if (["mp3", "m4a"].indexOf(files[i].format) > -1) { %>
+                <audio src="<%= $.cloudinary.url(files[i].public_id, { "version": files[i].version, "resource_type": 'video', "format": files[i].format }) %>" controls preload="<%= options.preload %>"/>
```

Comment:
`files[i].format` from an old commit
`preload="<%= options.preload %>"` allows the renderer to replace this with the `preload` value from `options`